amy/workshops/base_views.py

```diff
@@ -342,7 +343,7 @@ def autoresponder(self, form, fail_silently=True):
 
         try:
             email.send()
-        except SMTPException as e:
+        except (SMTPException, AnymailRequestsAPIError) as e:
```

Comment:
I wasn't able to reproduce this bug in tests, because Django was switching to a `locmem` email backend even if I explicitly enabled Anymail through environmental variables.

Thus I didn't include any regression tests for this bug.